addons/common/functions/fnc_isModLoaded.sqf

```diff
@@ -4,24 +4,25 @@
  * Check in CfgPatches if modification is loaded
  *
  * Arguments:
- * 0: Mod Name or Classname of the mod in CfgPatches <STRING>
+ * 0: Classname of the mod in CfgPatches <STRING>
  *
  * Return Value:
- * if modification is loaded <BOOL>
+ * If modification is loaded <BOOL>
  *
  * Example:
- * ["class"] call ace_common_fnc_isModLoaded
+ * "class" call ace_common_fnc_isModLoaded
  *
  * Public: Yes
  */
 
 params [["_modName", "", [""]]];
 
-private _return = GVAR(isModLoadedCache) get _modName;
+private _key = toLowerANSI _modName;
+private _return = GVAR(isModLoadedCache) get _key;
```

Comment:
```suggestion
GVAR(isModLoadedCache) getOrDefaultCall [toLowerANSI _modName, {isClass (configFile >> "CfgPatches" >> _key )}, true]
```

Reply:
I'll gladly use `getOrDefaultCall`, but I had the impression it wasn't liked. Is that true?

Reply:
I have nothing against it.